Sources/Clappr_iOS/Classes/Plugin/Core/JumpCorePlugin.swift

```diff
@@ -27,11 +27,11 @@ public class DoubleTapCorePlugin: DoubleTapPlugin {
         return pluginColidingWithGesture == nil
     }
     
-    @objc private func doubleTap(gestureRecognizer: UITapGestureRecognizer) {
+    @objc private func jump(gestureRecognizer: UITapGestureRecognizer) {
```

Comment:
What do you think about renaming this function to look like it was triggered by the gesture?
suggestion: `func didTap(gestureRecognizer: UITapGestureRecognizer)`
It will read like: `"Did tap gesture recognizer"` :+1:

Reply:
Done.